tests/go_test.go

```diff
@@ -1377,6 +1378,28 @@ func CheckStringIsNestedError(fail func(string, ...interface{})) {
 	b.CreateString("foo")
 }
 
+func CheckEmptiedBuilder(fail func(string, ...interface{})) {
+	f := func(strings []string) bool {
```

Comment:
As written this test doesn't use the arguments provided by quick check. Recommend something like [this](https://gist.github.com/invisiblefunnel/82fa6300282f5495c2ba5bda27bc9377).

Reply:
@invisiblefunnle I fixed them, thanks!

Reply:
@invisiblefunnle I fixed them, thanks!